csharp/ql/test/resources/stubs/_frameworks/Microsoft.AspNetCore.App/Microsoft.AspNetCore.Authentication.Core.cs

```diff
@@ -10,17 +9,15 @@ namespace Authentication
             public class AuthenticationFeature : Microsoft.AspNetCore.Authentication.IAuthenticationFeature
             {
                 public AuthenticationFeature() => throw null;
-                public Microsoft.AspNetCore.Http.PathString OriginalPath { get => throw null; set => throw null; }
-                public Microsoft.AspNetCore.Http.PathString OriginalPathBase { get => throw null; set => throw null; }
+                public Microsoft.AspNetCore.Http.PathString OriginalPath { get => throw null; set { } }
```

Comment:
Could also be `set => throw null;` to match all other callable bodies.

Reply:
I think I prefer to avoid `throw null` when possible.

Reply:
I have no preference other than consistency. Should `void` returning methods also have an empty block body?